packages/account/src/Components/article/article.jsx

```diff
@@ -21,8 +21,8 @@ const Article = ({ title, descriptions, onClickLearnMore }) => {
                         </Text>
                     ) : (
                         <ul className='da-article__list'>
-                            {descriptions.map((description, idx) => (
-                                <li key={idx}>
+                            {descriptions.map(description => (
+                                <li key={description?.component || description}>
```

Comment:
is this a react component?
if yess i think its better to not pass it as key

Reply:
Hello @ali-hosseini-fs ,

I'll see how to change this